snuba/web/split.py

```diff
@@ -277,13 +277,6 @@ def execute(
                     exc_info=True,
                 )
 
-        minimal_columns = {
-            (col.table_name, col.column_name)
-            for col in minimal_query.get_all_ast_referenced_columns()
-        }
-        if len(total_columns) <= len(minimal_columns):
-            return None
```

Comment:
This is not really needed anymore in this case, because after reducing the selected columns from the threshold to 3, the minimal query cannot reference more than the original query.